packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -45,7 +51,6 @@ export const createPatient = async (
   // validate facility exists and cx has access to it
   await getFacilityOrFail({ cxId, id: facilityId });
 
-  const requestId = uuidv7();
   const patientCreate: PatientCreate = {
```

Comment:
can you briefly explain why we are removing the request id?

Reply:
I've moved the patient discovery state into the HIE specific `externalData` maps as well as updated the patient discovery workflow functions (`discover`, `create`, `update`) to make requestId optional and while setting it explicitly if not passed.

Thus, we don't really need this shared request ID across HIEs in my opinion.

Reply:
But @Orta21 pointed out that we never set it explicitly anyway, so it should probably just be dropped as an optional argument and always set explicitly in `discover`, `create`, and `update`.

Reply:
My understanding is that we want to have a single request ID for anything that's happening within that API request (business operation).

In fact, we could leverage the "http" `reqId` we created for logging purposes and use that as the request ID, instead of creating a new UUIDv7 - it would make debugging easier, as one looks at the logs and knows the request ID or knowing the request ID is easy to filter log entries related to it.

Reply:
@Orta21 brought this up to -- using the "http" `reqId` makes sense to me, as this has value throughout the ecosystem. Creating a shared request ID via uuid7 simply for log analysis is a bit strange to me... creates the impression the code *needs* a shared request ID when the code itself does not (in my opinion).

Reply:
I've added `Interanl 1829` to the TODO in the code.